doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2451 @@
+
+You can index and slice NumPy arrays in the same ways you can slice Python lists.
+
+**Input:**
+
+::
+
+    data = np.array([1,2,3])
+
+    print(data[0])
+    print(data[1])
+    print(data[0:2])
+    print(data[1:])
+    print(data[-2:])
+
+**Output:**
+
+::
+
+  1
+  2
+  [1 2]
+  [2 3]
+
+You can visualize it this way:
+
+.. image:: images/np_indexing.png
+
+
+You may want to take a section of your array or specific array elements to use in further analysis or additional operations. To do that, you'll need to subset, slice, and/or index your arrays. 
+
+If you want to select values from your array that fulfill certain conditions, it's straightforward with NumPy. 
+
+For example, if you start with this array:
+
+::
+
+  a = np.array([[1 , 2, 3, 4], [5, 6, 7, 8], [9, 10, 11, 12]])
+
+You can easily print all of the values in the array that are less than 5.
+
+**Input:**
+
+::
+
+  print(a[a<5])
+
+**Output:**
+
+::
+  
+  [1 2 3 4]
+
+You can also select, for example, numbers that are equal to or greater than 5, and use that condition to index an array.
+
+**Input:**
+
+::
+
+  five_up = (a >= 5)
+  print(a[five_up])
+
+**Output:**
+
+::
+
+  [ 5  6  7  8  9 10 11 12]
+
+You can select elements that are divisible by 2:
+
+**Input:**
+
+::
+
+  divisible_by_2 = a[a%2==0]
+  print(divisible_by_2)
+
+**Output:**
+
+::
+
+  [ 2  4  6  8 10 12]
+
+Or you can select elements that satisfy two conditions using the ``&`` and ``|`` operators:
+
+**Input:**
+
+::
+
+  c = a[(a > 2) & (a < 11)]
+  print(c)
+
+**Output:**
+
+::
+
+  [ 3  4  5  6  7  8  9 10]
+
+While it would be incredibly inefficient for this array, you can also make use of the logical operators **&** and **|** in order to return boolean values that specify whether or not the values in an array fulfill a certain condition. This can be useful with arrays that contain names or other categorical values.
+
+**Input:**
+
+::
+
+  five_up = (array > 5) | (array == 5)
+  print(five_up)
+
+**Output:**
+
+::
+
+  [[False False False False]
+   [ True  True  True  True]
+   [ True  True  True  True]] 
+
+You can also use ``np.where()`` to select elements or indices from an array. 
+
+Starting with this array:
+
+::
+
+  a = np.array([[1 , 2, 3, 4], [5, 6, 7, 8], [9, 10, 11, 12]])
+
+You can use ``np.where()`` to print the indices of elements that are, for example, less than 5:
+
+**Input:**
+
+::
+
+  b = np.where(a<5)
+  print(b)
+
+**Output:**
+
+::
+
+  (array([0, 0, 0, 0]), array([0, 1, 2, 3]))
+
+In this example, a tuple of arrays was returned: one for each dimension. The first array represents the row indices where these values are found, and the second array represents the column indices where the values are found.
+
+If you want to generate a list of coordinates where the elements exist, you can zip the arrays, iterate over the list of coordinates, and print them. For example:
+
+**Input:**
+
+::
+
+  list_of_coordinates= list(zip(b[0], b[1]))
+
+  for cord in list_of_coordinates:
```

Comment:
`cord` --> `coord` would be a bit more standard

Reply:
Got it!